api/api/routers/mcp/mcp_server.py

```diff
@@ -688,97 +688,97 @@ async def search_documentation(
+     - 'organizations.mdx' - Documentation on team collaboration using organizations. Covers creating, joining, and switching between organizations, as well as managing members and join settings.
+     - 'glossary.mdx' - A glossary of key terms and concepts used throughout the WorkflowAI documentation. Provides definitions for important terminology.
+     - 'ask-ai.mdx' - Documentation for the 'Ask AI' assistant. Explains how to ask questions, use commands, and get guidance on using the platform.
+     - 'changelog.mdx' - A chronological record of updates, improvements, and new model integrations to the WorkflowAI platform.
+     - 'compliance.mdx' - Information on security and compliance. Covers SOC2 compliance, data handling policies, and provides answers to frequently asked questions about data privacy.
 
      **Use Cases:**
-     - 'use-cases/chatbot' - Use case guide for building a conversational AI chatbot. Covers basic setup, conversation management, streaming, error handling, and deployment.
-     - 'use-cases/new_agent' - Comprehensive framework for building AI agents from scratch. Covers requirements analysis, agent types, model selection, prompt design, evaluation methods, and best practices.
-     - 'use-cases/migrating_existing_agent' - Step-by-step guide for migrating existing AI agents to the WorkflowAI platform. Covers understanding your current agent, data requirements, and migration best practices.
-     - 'use-cases/improving_and_debugging_existing_agent' - Guide for troubleshooting and optimizing existing AI agents. Covers common errors like max_tokens_exceeded, debugging with metadata, and performance optimization strategies.
-     - 'use-cases/classifier' - Use case guide for building AI classifiers. Covers techniques for text categorization, sentiment analysis, and more, from basic to advanced implementations.
-     - 'use-cases/image-input' - Use case guide for using images as input for AI agents. Covers processing and analyzing images to extract information, answer questions, and perform vision-based tasks.
-     - 'use-cases/pdf-input' - Use case guide for processing and extracting information from PDF documents. Covers techniques for using PDF files as input for summarization, data extraction, and question-answering tasks.
-     - 'use-cases/image-generation' - Use case guide for generating and editing images. Covers best practices for prompting, customizing image outputs, and using the WorkflowAI SDK for image generation tasks.
-     - 'use-cases/mcp' - A collection of use-case scenarios for AI agents that interact with the WorkflowAI platform. Outlines the required capabilities for agents to perform tasks like optimization, debugging, and deployment.
+     - 'use-cases/chatbot.mdx' - Use case guide for building a conversational AI chatbot. Covers basic setup, conversation management, streaming, error handling, and deployment.
```

Comment:
Why do we need the `.mdx` ? That seems confusing. Can we auto add if needed in local mode instead ?

Reply:
done https://github.com/WorkflowAI/WorkflowAI/pull/565/commits/80c58c9400fa03dedc49e466cc40652a46ed3f5b